_posts/2023-02-20-intelligent-automation.md

```diff
@@ -0,0 +1,223 @@
+In addition to this, the same level of transparency (such as for audit or privacy purposes) should also apply to the bots. 
+This way correct optimization of the process automation can be determined.
+
+These different components are aptly assembled in Nathaniel Palmer’s vision for future process automation solutions:
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2023-02-20-intelligent-automation/ibpms.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+The three guiding principles are Rules, Robots, and Relationships. 
+The Robot part of the equation was already tackled earlier in this post. 
+The Relationship part focuses mainly on the data used in the processes. 
+There is a shift here from the problems of storing and replication data locally towards the problems of where to get data in the wide world. 
+The different pieces of the information puzzle could be spread across dozens of repositories all over the world, and although our complete information metamodel still exists, its parts need to be fetched from each of these repositories to complete any sort of 360° view.
+
+Rules are formed by automating decisions and letting AI solutions approach the available information with machine learning to perform both analytical and predictive analyses. 
+This isn’t magic, however, and these AI need to be properly trained using historical data to perform with any level of adequacy. 
+These bots were dubbed probabilistic as opposed to the deterministic nature of RPA bots. 
+
+With all these different angles to process automation, it might look like a mire of patchwork islands. 
+But this is where the traditional iBPMS excels, bridging these islands to arrive at an end-to-end process, managing the sequencing, and overseeing the state of running processes. 
+This eventually leads to the promise of intelligent automation: Expanding the efficiency of automation while delivering greater transparency and policy compliance. 
+And herein lies the business value of this proposition. 
+If traditional BPMS automation proves too rigid for the task, consider splitting up the end-to-end process into process fragments to become more flexible. 
+Recreating this end-to-end process from these fragments can be done by structuring them through a case that indicates the major phases in the process. 
+As such these fragments become once again linked, but still retain a dynamic and adaptable nature. 
+This is the intentional process as posited by the people from Flowable.
+
+It is easy to have initiatives for intelligent automation derail, but here we can apply the same lessons we learned for process management adoption:
+1.	Make everyone (on every level) in the organization aware of the benefits and how it will help them in their jobs (not replace them). It alleviates tedious repetitive work and frees up time for employees to pursue more worthwhile endeavors within their organization.  
```

Comment:
```suggestion
1.	Make everyone, on every level, in the organization aware of the benefits and how it will help them in their jobs, not replace them. It alleviates tedious repetitive work and frees up time for employees to pursue more worthwhile endeavors within their organization.  
```